common/remote/sockfile.cpp

```diff
@@ -4358,7 +4358,14 @@ class CRemoteIndexCountActivity : public CRemoteIndexBaseActivity
 
         // create helper
         inMeta.setown(getTypeInfoOutputMetaData(config, "input", false));
-        outMeta.set(inMeta);
+
+        // create a { unsigned8 } output meta for the count
+        const RtlIntTypeInfo countFieldType(type_int, 8);
+        const RtlFieldStrInfo countField("count", nullptr, &countFieldType);
+        const RtlFieldInfo * const fields[2] = { &countField, nullptr };
+        const RtlRecordTypeInfo record = RtlRecordTypeInfo(type_record, 2, fields);
```

Comment:
These need to be static const objects - otherwise they will go out of scope and be destroyed, but outMeta will still exist.